com.alizadicomviewer.AlizaMS.metainfo.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id type="desktop">com.alizadicomviewer.AlizaMS</id>
```

Comment:
`alizadicomviewer.com` doesn't exist.

The proper ID should probably be `io.github.AlizaMedicalImaging.AlizaMS`

Reply:
It is https://www.aliza-dicom-viewer.com/

Anyway. I [asked upstream what they think](https://github.com/AlizaMedicalImaging/AlizaMS/issues/5#issuecomment-993624931)
And indeed io.github.AlizaMedicalImaging.AlizaMS is better.

Should I rename the git repository too? (Ehm, is it possible? Or is it more simple to start from scratch?)

Thanks

Reply:
Renaming the files (and changing the ID) is all you need.

`com.aliza_dicom_viewer.AlizaMS` could work too. I just could not easily find the link ot their website.